src/components/Canvas/Canvas.tsx

```diff
@@ -73,8 +73,8 @@ export function Canvas(props: {
     canvas.height = props.height;
 
     const field = new Field(
-      canvas.width,
-      canvas.height,
+      props.width,
+      props.height,
```

Comment:
@ivkazanovskiy  Did I understand correctly when you said to use the values from the `props` for the field instead of pulling them directly from the `canvas`?

Reply:
@bnzone yes, you are right. 
Nevertheless, In my opinion we have to get rid of this code too:
``` JS
canvas.width = props.width;
canvas.height = props.height;
```

There is a crucial difference between the canvas element and the Field.
Canvas is a part of webpage and it size should be defined with CSS.
At the same time, Field's settings should be changes according to user's wishes.

For example:
According to UX/UI the canvas always has full window size.
But a user wants to create big Field 1500*1000 for big dungeon quest or small Field 50*30 for a tavern scene.